apps/asap-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -323,6 +323,41 @@ export const getResearchOutputEvent = (): WebhookPayload<ResearchOutput> => ({
   },
 });
 
+export const researchOutputEvent = (
```

Comment:
this is how fixtures become cluttered - you have an almost identical fixture above - if you want to build another one which is more generic why didn't you remove the one above and replace its references?

Reply:
no problem - now that you converted it I would recommend that you follow our current practices and add a `get` prefix for the fixture generator because essentially this is a function that returns a fixture, not the fixture itself